openstack/imageservice/v2/images/urls.go

```diff
@@ -0,0 +1,68 @@
+package images
+
+import (
+	"strings"
+
+	"github.com/rackspace/gophercloud"
+)
+
+// `listURL` is a pure function. `listURL(c)` is a URL for which a GET
+// request will respond with a list of images in the service `c`.
+func listURL(c *gophercloud.ServiceClient) string {
+	return c.ServiceURL("images")
+}
+
+func createURL(c *gophercloud.ServiceClient) string {
+	return c.ServiceURL("images")
+}
+
+// `imageURL(c,i)` is the URL for the image identified by ID `i` in
+// the service `c`.
+func imageURL(c *gophercloud.ServiceClient, imageID string) string {
+	return c.ServiceURL("images", imageID)
+}
+
+// `getURL(c,i)` is a URL for which a GET request will respond with
+// information about the image identified by ID `i` in the service
+// `c`.
+var getURL = imageURL
```

Comment:
Instead of these `var`s, let's be explicit for consistency with other services like here: https://github.com/rackspace/gophercloud/blob/master/openstack/compute/v2/servers/urls.go